docs/release-source/release/sandbox.md

```diff
@@ -127,19 +128,86 @@ A convenience reference for [`sinon.assert`](./assertions)
 
 *Since `sinon@2.0.0`*
 
+#### `sandbox.replace(object, property, replacement);`
+
+Replaces `property` on `object` with `replacement` argument.
+
+`replacement` can be any value, including `spies`, `stubs` and `fakes`.
+
+This method only works on non-accessor properties, for replacing accessors, use `sandbox.replaceGetter()` and `sandbox.replaceSetter()`.
```

Comment:
This is great idea btw.

Since we've been dealing with `getters/setters` we've had a lot of trouble making the APIs generic enough to deal with them, but I guess that encapsulating that logic into separate methods is a lot better and allows us to avoid having lots of special conditions and a complex control-flow.